linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml

```
@@ -31,7 +31,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-010270
-    cis: 5.3.3
+    cis@rhel7: 5.3.3
```

The `rhel7`reference is `5.3.4.
```suggestion
    cis@rhel7: 5.3.4
```

---

Are you sure? The `accounts_password_pam_unix_remember` rule sets `pam_pwhistory` which is under `5.3.3` in RHEL7 benchmark, the `5.3.4` as you suggested sets password encryption to sha512.

---

You are correct, :)